site/source/tutorials/005_form_widgets/demo/initial/biz-e-corp/src/widgets/Worker.ts

```diff
@@ -0,0 +1,102 @@
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+import { DNode, WidgetProperties } from '@dojo/widget-core/interfaces';
+import { v } from '@dojo/widget-core/d';
+import { theme, ThemeableMixin } from '@dojo/widget-core/mixins/Themeable';
+import * as css from '../styles/worker.css';
+
+export interface WorkerProperties extends WidgetProperties {
+	firstName?: string;
+	lastName?: string;
+	email?: string;
+	timePerTask?: number;
+	tasks?: string[];
+}
+
+const WorkerBase = ThemeableMixin(WidgetBase);
+
+@theme(css)
+export default class Worker extends WorkerBase<WorkerProperties> {
+	private _isFlipped = false;
+
+	protected render(): DNode {
+		return v('div', {
+			classes: this.classes(css.worker, this._isFlipped ? css.reverse : null)
+		}, [
+			this._renderFront(),
+			this._renderBack()
+		]);
+	}
+
+	private _renderFront(): DNode {
+		const {
+			firstName = 'firstName',
+			lastName = 'lastName'
+		} = this.properties;
+
+		return v('div', {
+			classes: this.classes(css.workerFront),
+			onclick: this.flip
+		}, [
+			v('img', {
+				classes: this.classes(css.image),
+					src: 'images/worker.svg' }, []),
+				v('div', [
+					v('strong', [ `${lastName}, ${firstName}` ])
+				])
+			]
+		);
+	}
+
+	private _renderBack(): DNode {
+		const {
+			firstName = 'firstName',
+			lastName = 'lastName',
+			email = 'unavailable',
+			timePerTask = 0,
+			tasks = []
+		} = this.properties;
+
+		return v('div', {
```

Comment:
Does the indentation here match the style guide (in this block of code), not this one line